subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/CancellationSpec.groovy

```diff
@@ -87,14 +87,19 @@ latch.await()
         targetVersion >= GradleVersion.version('5.1')
     }
 
+    private static void verifyBuildCancelledExceptionMessage(TestResultHandler resultHandler) {
+        // https://github.com/gradle/gradle-private/issues/1760
+        assert resultHandler.failure.cause.message in ["Build cancelled.", "Daemon was stopped to handle build cancel request."]
+    }
+
```

Comment:
:o: I would move the function below both usages for readability

Reply:
Done.